modules/nf-core/rtn/tna/main.nf

```diff
@@ -0,0 +1,50 @@
+process RTN_TNA {
+    debug true
+    tag "{$tni_object.name}"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+	'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/96/96979cd0715edeb5d68ebbd19a353760298ebf53bf729d3b68764b2bb00683f7/data':
+        'community.wave.seqera.io/library/bioconductor-rtn:2.30.0--71b797cd8b2d56b3' }"
+
+    input:
+    tuple val(meta), path(tni_object)
+    tuple val(meta), path(degs)
```

Comment:
I think that the best practice is to give a different name such as `meta2`, `meta3`, etc. because you can't guarantee that all metadata channels will be identical.